documentation/disaster-recovery.md

```diff
@@ -103,25 +107,25 @@ In case the database instance is lost, the objectives are:
 
 ### Recreate the lost postgres database instance
 
-In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
+Please note, this process could take about circa 60 min to compleate. In case the database service is deleted or in an inconsistent state we must recreate it and repopulate it.
```

Comment:
I think about and circa mean pretty much the same thing, so you can drop the circa.

Reply:
If the database service has been deleted or is in an inconsistent state we must recreate it and repopulate it.